pkg/ui/distoss/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+genrule(
+    name = "assets",
```

Comment:
Same comments as above: should have distinct `name`, directory output is unsound, use `cp -R` instead of `cp -r`

Reply:
Done